### PR TITLE
add tests for modal

### PR DESCRIPTION
**Pull request by jask1m**
Close Issue #1440

### CODE REVIEW

File: test/frontend/DecisionModal.test.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  const modalProps = {
+    headerText: 'This the test header',
+    bodyText: 'test body',
+  }
+  const wrapper = mount(<ConfirmationModal {...modalProps} />);
+  it('Should render the headerText prop in a h3 tag', () => {
+    expect(wrapper.find('h3').text()).to.equal('This the test header');
+  });
+  it('Should render the bodyText prop in a p tag', () => {
+    expect(wrapper.find('p').text()).to.equal('test body');
```
Comment (evanugarte) [inline test/frontend/DecisionModal.test.js]:
```suggestion
    expect(wrapper.find('h3').text()).to.equal(modalProps.headerText);
  });
  it('Should render the bodyText prop in a p tag', () => {
    expect(wrapper.find('p').text()).to.equal(modalProps.bodyText);
```

**Review by evanugarte — APPROVED** [review]

**Review by weslayer — APPROVED** [review]
very nice

File: test/frontend/DecisionModal.test.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    bodyText: 'test body',
+  }
+  const wrapper = mount(<ConfirmationModal {...modalProps} />);
+  it('Should render the headerText prop in a h3 tag', () => {
+    expect(wrapper.find('h3').text()).to.equal(modalProps.headerText);
+  });
+  it('Should render the bodyText prop in a p tag', () => {
+    expect(wrapper.find('p').text()).to.equal(modalProps.bodyText);
+  });
+  it("Should render 'Confirm' when no prop is passed", () => {
```
Comment (evanugarte) [inline test/frontend/DecisionModal.test.js]:
```suggestion
  it('Should render \'Confirm\' when no prop is passed', () => {
```

File: test/frontend/DecisionModal.test.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+import ConfirmationModal from '../../src/Components/DecisionModal/ConfirmationModal';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe('<ConfirmationModal />', () => {
+  const modalProps = {
+    headerText: 'This the test header',
+    bodyText: 'test body',
+  }
```
Comment (evanugarte) [inline test/frontend/DecisionModal.test.js]:
```suggestion
  };
```

File: test/frontend/DecisionModal.test.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    expect(wrapper.find('h3').text()).to.equal(modalProps.headerText);
+  });
+  it('Should render the bodyText prop in a p tag', () => {
+    expect(wrapper.find('p').text()).to.equal(modalProps.bodyText);
+  });
+  it("Should render 'Confirm' when no prop is passed", () => {
+    const wrapper = mount(<ConfirmationModal />);
+    expect(wrapper.find('button').at(0).text()).to.equal('Confirm');
+  });
+  it("Should render 'Cancel' when no prop is passed", () => {
```
Comment (evanugarte) [inline test/frontend/DecisionModal.test.js]:
```suggestion
  it('Should render \'Cancel\' when no prop is passed', () => {
```

File: test/frontend/DecisionModal.test.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  });
+  it('Should render \'Confirm\' when no prop is passed', () => {
+    const wrapper = mount(<ConfirmationModal />);
+    expect(wrapper.find('button').at(0).text()).to.equal('Confirm');
+  });
+  it('Should render \'Cancel\' when no prop is passed', () => {
+    const wrapper = mount(<ConfirmationModal />);
+    expect(wrapper.find('button').at(1).text()).to.equal('Cancel');
+  });
+});
```
Comment (evanugarte) [inline test/frontend/DecisionModal.test.js]:
```suggestion
});

```

**Pull request merged**